applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -387,3 +387,87 @@ Test command
 ------------
 
 The test command is not supported.
+
+SLM CMNG #XCMNG
+===============
+
+The ``#XCMNG`` command manages the credentials to support :ref:`CONFIG_SLM_NATIVE_TLS <CONFIG_SLM_NATIVE_TLS>`.
+This command is implemented similarly to the modem ``%CMNG`` command.
+
+Set command
+-----------
+
+The set command is used for credential storage management.
+The command writes, reads, deletes, and checks the existence of keys and certificates.
+
+Syntax
+~~~~~~
+
+The following is the syntax when :ref:`CONFIG_SLM_NATIVE_TLS <CONFIG_SLM_NATIVE_TLS>` is selected:
+::
+
+   #XCMNG=<opcode>[,<sec_tag>[,<type>[,<content>]]]
+
+The ``<opcode>`` parameter is an integer.
+It accepts the following values:
+
+* ``0`` - Write a credential.
+* ``1`` - List credentials (currently not supported).
+* ``2`` - Read a credential (currently not supported).
+* ``3`` - Delete a credential.
+
+The ``<sec_tag>`` parameter is an integer ranging between ``0`` and ``2147483647``.
+It is mandatory for *write*, *read*, and *delete* operations.
+It is optional for *list* operations.
+
+The ``<type>`` parameter is an integer.
+It accepts the following values:
+
+* ``0`` - Root CA certificate (ASCII text)
+* ``1`` - Certificate (ASCII text)
+* ``2`` - Private key (ASCII text)
+
+The ``<content>`` parameter is a string.
+It is mandatory if ``<opcode>`` is ``0`` (write a credential).
+An empty string is not allowed.
+A Privacy Enhanced Mail (PEM) file enclosed in double quotes (X.509 PEM entities).
```

Comment:
@junqingzou What is the meaning of this sentence?

Reply:
AT#XCMNG is re-implement of AT%CMNG modem command.
The text is from modem command manual:
https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/security/cmng_set.html?cp=2_1_11_7_0